src/java.base/share/classes/module-info.java

```diff
@@ -182,6 +182,9 @@
         jdk.compiler,
         jdk.jlink;
     exports jdk.internal.logger to
+/*[IF OPENJCEPLUS_SUPPORT]*/
+        openjceplus,
+/*[ENDIF] OPENJCEPLUS_SUPPORT */
```

Comment:
It should be a priority to eliminate the use of internal classes by OpenJCEPlus.

Reply:
@keithc-ca 
Yes, we are working on the elimination of the use of the internal classes by OpenJCEPlus. But for the current phase, we still have to use those internal classes. 
@jasonkatonica FYI.

Reply:
The team faced a decision of maintaining 100k lines of code that was a duplicate of the sun internal classes or use some of the sun internal classes. A decision was made to use the sun internal classes. We are however in the long run planning on trying to find alternates and more and more external functions that could be used as an equivalent wherever possible. 

This particular package is something we certainly plan on removing as there are many alternatives to the jdk.internal.logger package. It will however not be done in this phase of development.